spec/requests/providers/proceedings_types_spec.rb

```diff
@@ -140,8 +133,6 @@
         let(:add_proceeding_service) { double(LegalFramework::AddProceedingService, call: false) }
 
         before do
-          allow(LegalFramework::ProceedingTypesService).to receive(:new).with(legal_aid_application).and_return(proceeding_type_service)
```

Comment:
Can the LegalFramework::ProceedingTypesService be removed too?
Is it in a separate Ticket or is it still needed?

Reply:
It can be yes, but we can get rid of that in the general tidy up, which I think is all there is left to do on this apart from dropping the tables